### ISSUE TITLE
misleading statement in the manual

### ISSUE THREAD

**Issue by oguz-ismail** [stedolan/jq#2119]
The manual states:
> jq has a few operators of the form **a op= b**, which are all equivalent to **a |= . op b**.

However this is not true; `a op= b` is not always equal to `a |= . op b` in function. For instance, when the RHS of `op=` is a stream of `n` values, the result is `n` values; in the same situation, `|= . op` yields only one value (due to `., break $out` in the definition of `_modify`). So, that statement is misleading.

Compare these:
```
$ jq -cn '{foo:1} | .foo += (1,2,3)'
{"foo":2}
{"foo":3}
{"foo":4}
$
$ jq -cn '{foo:1} | .foo |= . + (1,2,3)'
{"foo":2}
```
And these two:
```
$ jq -c '.foo[].bar += input' <<EOD
{"foo":[{"bar":1},{"bar":2}]}
3
4
EOD
{"foo":[{"bar":4},{"bar":5}]}
jq: error (at <stdin>:3): break
$
$ jq -c '.foo[].bar |= . + input' <<EOD
{"foo":[{"bar":1},{"bar":2}]}
3
4
EOD
{"foo":[{"bar":4},{"bar":6}]}
```

**Comment by emanuele6** [comment]
Yes, it should say they are equivalent to `(b as $b | a |= . op $b)`.
